src/lib/dialog/dialog-config.ts

```diff
@@ -69,5 +69,14 @@ export class MdDialogConfig {
   /** ID of the element that describes the dialog.  */
   ariaDescribedBy?: string | null = null;
 
+  /** Aria label to assign to the dialog element */
+  ariaLabel?: string | null = null;
+
+  /**
+   * Selector for an element to be focused when the dialog is opened. If omitted or the
+   * element is not found, the dialog will focus the first focusable element.
+   */
+  focusOnOpen?: string | null = null;
```

Comment:
My reasoning to go for a selector instead of the boolean was to make it more flexible so we don't have to tweak it if somebody wants to focus something different from the container. E.g. this allows for the selector to be constructed at runtime, whereas the focus trap attribute needs to be set in advance.